images/twitter.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="twitter.css">
+    <title>twitter clone</title>
+</head>
+
+<body>
+    <div class="bigcontainer">
+
+        <div class="leftcontainer">
+
+            <div class="twitterlogo">
+                <img src="https://www.svgrepo.com/show/183608/twitter.svg" alt="logoicon">
```

Comment:
this is good. I still haven't succeeded in managing to get the svgs into an img. The way you've done it makes your code really tidy